sycl/test-e2e/BFloat16/bfloat16_example.cpp

```diff
@@ -13,16 +13,16 @@
 // RUN: %clangxx -fsycl -fsycl-targets=spir64 %s -o %t.out
 // RUN: %{run} %t.out
 
-// RUN: %clangxx -fsycl -fsycl-targets=spir64_gen -Xsycl-target-backend "-device gen9" %s -o %t.out
+// RUN: %clangxx -fsycl -fsycl-targets=spir64_gen -Xsycl-target-backend "-device gen12lp" %s -o %t.out
 // RUN: %{run} %t.out
```

Comment:
Seems to me like this is GPU AOT, so it should be limited to only run on GPU.
```suggestion
// RUN: %if gpu %{ %{run} %t.out %}
```